hubtty/sync.py

```diff
@@ -676,7 +673,6 @@ def _syncChange(self, sync):
             remote_commits[-1]['_hubtty_remote_checks_data'] = remote_checks_data
 
         fetches = collections.defaultdict(list)
-        parent_commits = set()
```

Comment:
This was picked because of the commented code below for which I'm not exactly sure what it's supposed to do. We should either remove everything including the commented out code or comment out this line until we decide if it's needed or not.

https://github.com/hubtty/hubtty/blob/cb67188112c7ba76e366a622b1fad027736ee78b/hubtty/sync.py#L757-L765